$NetBSD$

* Fix build. Use devel/libltdl instead of internal libltdl.
- Build dynamic modules only dynamic (-shared), not also static.

--- Makefile.am.orig	2019-09-10 18:06:20.000000000 +0000
+++ Makefile.am
@@ -11,7 +11,7 @@ extpkg_headers = @EXTPKG_HEADERS@
 extpkg_ldadd   = @EXTPKG_LDADD@
 hqa_inc        = @HQA_INC@
 lns            = @LN_S@
-ldadd          = @LIBS@
+ldadd          = @LIBS@ -lltdl
 
 #------------------------------------------------------------------------------
 
@@ -140,9 +140,9 @@ dyndev_SRC =  \
 #                    to Windows DLLs
 #-----------------------------------------------------------------
 
-XSTATIC =
+XSTATIC = -shared
 DYNSRC  =
-LTDL    =  ltdl.c
+LTDL    =
 
 DYNMOD_LD_FLAGS =   \
   -module           \
@@ -500,7 +500,6 @@ libherc_la_SOURCES = \
   $(DYNSRC)
 
 EXTRA_libherc_la_SOURCES = \
-  ltdl.c             \
   memrchr.c          \
   $(dynamic_SRC)     \
   $(dyndev_SRC)      \
@@ -939,7 +938,6 @@ noinst_HEADERS =          \
   inline.h                \
   linklist.h              \
   logger.h                \
-  ltdl.h                  \
   machdep.h               \
   memrchr.h               \
   mpc.h                   \
